ci/Dockerfile.ubuntu.20.04.amdgpu

```diff
@@ -0,0 +1,80 @@
+# Taichi Dockerfile for development
+FROM rocm/dev-ubuntu-20.04:5.2
+
+LABEL maintainer="https://github.com/taichi-dev"
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common && \
+    apt-get install -y libtinfo-dev \
+                       clang-10 \
+                       wget \
+                       git \
+                       unzip \
+                       ffmpeg \
+                       libxrandr-dev \
+                       libxinerama-dev \
+                       libxcursor-dev \
+                       libxi-dev \
+                       libglu1-mesa-dev \
+                       freeglut3-dev \
+                       libssl-dev \
+                       libglm-dev \
+                       libxcb-keysyms1-dev \
+                       libxcb-dri3-dev \
+                       libxcb-randr0-dev \
+                       libxcb-ewmh-dev \
+                       libpng-dev \
+                       g++-multilib \
+                       libmirclient-dev \
+                       libwayland-dev \
+                       bison \
+                       libx11-xcb-dev \
+                       liblz4-dev \
+                       libzstd-dev \
+                       qt5-default \
+                       libglfw3 \
+                       libglfw3-dev \
+                       libjpeg-dev
+# Install the latest version of CMAKE v3.20.5 from source
+WORKDIR /
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.20.5/cmake-3.20.5-linux-x86_64.tar.gz
+RUN tar xf cmake-3.20.5-linux-x86_64.tar.gz && \
+    rm cmake-3.20.5-linux-x86_64.tar.gz
+ENV PATH="/cmake-3.20.5-linux-x86_64/bin:$PATH"
+
+# Install LLVM 15
+WORKDIR /
+# Make sure this URL gets updated each time there is a new prebuilt bin release
+RUN wget https://github.com/GaleSeLee/assets/releases/download/v0.0.1/taichi-llvm-15.0.0-linux.zip
```

Comment:
Please use @ailzhang 's version, grep `taichi-llvm` from `.github` directory to find the link.

Reply:
We use a different target of llvm. This is for `x86` and `amdgpu` while `taichi-llvm` from `.github` is for `x86` and `nvptx`